commit 51202798862c79ad417936cfe2d1847b1da4ab87
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Mon Oct 1 13:06:42 2012 +0100

    uxa: Remove USE_GLAMOR check from unrealize glyph caches
    
    In an interesting twist during teardown, the driver info structure is
    freed long before the caches. However, the glyph cache teardown was
    checking to see if glamor was enabled to see if it could skip the
    teardown. In fact, since we won't have created the glyph caches in that
    circumstances it was safe any way.
    
    Reported-by: Maarten Lankhorst <m.b.lankhorst@gmail.com>
    Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>

diff --git a/uxa/uxa-glyphs.c b/uxa/uxa-glyphs.c
index 527942a..f5d4d08 100644
--- a/uxa/uxa-glyphs.c
+++ b/uxa/uxa-glyphs.c
@@ -112,9 +112,6 @@ static void uxa_unrealize_glyph_caches(ScreenPtr pScreen)
 	uxa_screen_t *uxa_screen = uxa_get_screen(pScreen);
 	int i;
 
-	if (uxa_screen->info->flags & UXA_USE_GLAMOR)
-		return;
-
 	if (!uxa_screen->glyph_cache_initialized)
 		return;
 
